src/listeners/autoReactHelperFunctions.ts

```diff
@@ -1,13 +1,14 @@
 import { Client, Message } from "discord.js"
 
-const greetings = ['hallo', 'hi', 'hey', 'moin', 'servus', 'guten morgen', 'guten tag', 'guten abend', 'wuhu', 'nabend']
+const greetings = ['Hallo', 'Hi', 'Hey', 'Moin', 'Moinsen', 'Servus', 'Guten Morgen', 'Guten Tag', 'Guten Abend', 'Wuhu', 'Nabend']
 
 const sleepings = ['gute nacht', 'nachti', 'schlaft gut', 'gn8']
 
 function containsKeywordFromArray(msg: string, keywords: string[]): boolean {
   const msgLower = msg.toLowerCase()
-  for (const keyword of keywords) {
+  for (let keyword of keywords) {
     // check if the msg includes the keyword
+    keyword = keyword.toLowerCase()
```

Comment:
i dont really get why this is better xD

Reply:
ah. got it

Reply:
nevermind, makes sense

Reply:
I changed the Code when the Bot greets a User. It chooses a random Greeting from the Greetings Array. If the values of the Greetings Array are lower case it looks weird. German sentences start with an Uppercase Letter.